src/main/java/com/exactpro/th2/common/schema/filter/strategy/impl/AbstractFilterStrategy.java

```diff
@@ -62,22 +63,24 @@ private boolean checkValues(Map<String, String> messageFields, MultiValuedMap<St
         });
     }
 
-    private boolean checkValue(String value1, FieldFilterConfiguration filterConfiguration) {
-        if (StringUtils.isEmpty(value1)) {
+    private boolean checkValue(String value, FieldFilterConfiguration filterConfiguration) {
+        if (StringUtils.isEmpty(value)) {
             return false;
```

Comment:
Why are we returning `false` if the value is empty? What about the EMPTY filter?